applications/nrf_desktop/README.rst

```diff
@@ -1081,6 +1082,49 @@ At this point, you can start using the new sensor by completing the following st
 #. Enable the new sensor driver.
 #. Select the new sensor driver in the application configuration options.
 
+Changing interrupt priority
```

Comment:
This is very nice content that I think should be placed more prominently than the README for this application. Perhaps a top-level user guide? @ru-fu  / @carlescufi ?

Reply:
Yes, this definitely belongs elsewhere. But we don't have a home right now, so for 1.3.0 this will need to live here.